package.json

```diff
@@ -1,78 +1,52 @@
 {
-  "name": "@ngui/auto-complete",
-  "version": "2.0.0",
-  "description": "Angular Input Autocomplete",
-  "license": "MIT",
-  "main": "dist/auto-complete.umd.js",
-  "module": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "name": "auto-complete",
+  "version": "0.0.0",
   "scripts": {
-    "start": "webpack-dev-server --port 9002 --content-base app --config app/webpack.config --open",
-    "lint": "npm run tslint 'src/**/*.ts' 'app/**/*.ts'",
-    "clean": "rimraf dist",
-    "build": "npm-run-all --serial lint clean build:ngc build:umd build:app",
-    "build:ngc": "ngc -p tsconfig.ngc.json",
-    "build:umd": "webpack --mode development",
-    "build:app": "webpack --mode production --config app/webpack.config",
-    "build-and-publish": "npm run build && npm version patch && npm publish --access=public",
-    "tslint": "tslint"
+    "ng": "ng",
+    "start": "ng serve",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e",
+    "build-lib:watch": "ng build auto-complete --watch",
+    "build-lib:prod": "ng build auto-complete --prod"
   },
+  "private": true,
   "dependencies": {
-    "@ngui/utils": "^0.8.1"
+    "@angular/animations": "~9.0.0",
+    "@angular/cdk": "^9.1.0",
+    "@angular/common": "~9.0.0",
+    "@angular/compiler": "~9.0.0",
+    "@angular/core": "~9.0.0",
+    "@angular/forms": "~9.0.0",
+    "@angular/material": "^9.1.0",
+    "@angular/platform-browser": "~9.0.0",
+    "@angular/platform-browser-dynamic": "~9.0.0",
+    "@angular/router": "~9.0.0",
+    "rxjs": "~6.5.4",
+    "tslib": "^1.10.0",
+    "zone.js": "~0.10.2"
```

Comment:
There are a lot of dependencies, you need to move them to dev if you want them only for dev, anything optional "and not a must" should be peer, like material, dependencies here for what required to get this works.

Reply:
`/auto-complete/projects/auto-complete/package.json`

Please check this `package.json`. Peer dependencies are listed there. The current file is for the base.

Reply:
It should be fixed. But refer - `auto-complate/projects/auto-complete/package.json` file.